src/devices/moes.ts

```diff
@@ -182,6 +193,12 @@ const definitions: DefinitionWithExtend[] = [
                     .withFeature(exposesLocal.minute('sunday_p4_minute'))
                     .withFeature(exposesLocal.program_temperature('sunday_p4_temperature')),
             ];
+
+            if (device?.manufacturerName != '_TZE200_aoclfnxz') {
+                arr.splice(2, 0, e.deadzone_temperature());
+            }
```

Comment:
don't like this solution. I don't know about other devices but this one doesn't have this expose.

Reply:
Can't you do:

```
if (device?.manufacturerName !== '_TZE204_aoclfnxz') {
                arr.push(e.deadzone_temperature());
            }
```

and remove `e.deadzone_temperature()` from `arr`

Reply:
It's easy but then it will be the last in the gui list?
As I understand fields placed in the order they will be presented in views?

Reply:
Made as insert instead of removing.

Reply:
Yes please do the insert instead of removing.

Reply:
I've done it already

`arr.splice(3, 0, e.min_temperature_limit());`